webauthn/helpers/base64url_bytes.py

```diff
@@ -0,0 +1,40 @@
+from base64 import urlsafe_b64decode
+from typing_extensions import Annotated
+
+from pydantic import (
+    AfterValidator,
+    BeforeValidator,
+    FieldValidationInfo,
+    PlainSerializer,
+    WithJsonSchema,
+)
+from pydantic_core import PydanticCustomError
+
+from webauthn.helpers import bytes_to_base64url
+
+
+def validate_base64url(value: bytes, info: FieldValidationInfo) -> bytes:
+
+    if info.mode == "json":
+        try:
+            return urlsafe_b64decode(value + b"====")
+        except ValueError as e:
+            raise PydanticCustomError(
+                "base64_decode", "Base64 decoding error: '{error}'", {"error": str(e)}
```

Comment:
I'd like to try and remove as much ambiguity as to whether base64 or base64url is in play. I specifically chose base64url for everything because the spec declares it as a dependency:

https://www.w3.org/TR/webauthn-2/#sctn-dependencies

Thus in places like here I want care to be taken to ensure that we're using base64url through and through.

How about something like this instead:

```suggestion
                "base64url_decode", "Base64URL decoding error: '{error}'", {"error": str(e)}
```